TERMSOFUSE.txt

```diff
@@ -0,0 +1,14 @@
+Terms of Use
+
+Stanford University owns all Stanford-branded themes and governs their use to benefit academic, research, communication and other University activities and interests. The University has the right to protect the use of these themes, the University name and University identity marks such as its seal, logos and symbols in all situations.
+
+Use of the Stanford-branded themes is limited to Stanford University academic and administrative activities. Use is granted solely to:
+
+* University academic departments, centers and research groups;
+* University administrative units;
+
+The Stanford brand bar and Stanford universal footer should not be edited or altered in any way, following the Identity toolkit guidelines at https://identity.stanford.edu/web-toolkit/
```

Comment:
I think the correct url is https://identity.stanford.edu/web-mobile.html#build-something-new. Not sure about including the anchor, tho.